Typos and wording (#291)
johko/computer-vision-course: chapters/en/unit1/chapter1/applications.mdx

    <img src="https://huggingface.co/datasets/hf-vision/course-assets/resolve/743a2a115b53f258c9e6bc7744534d9e03b8a124/ethical_considerations.png" alt="Another summary of Ethical Considerations. Created Isabella Bicalho-Frazeto and Seshu Pavan">
</div>

> Ethical considerations in computer vision are of paramount importance as this technology becomes increasingly integrated into various aspects of our lives. It is important to note that ethical consideration existed long before the spread of AI based technologies. It is intrinsically tied to its birth.

Reviewer: ```suggestion
Ethical considerations in computer vision are increasingly important technologies becomes increasingly integrated into various aspects of our lives. It is important to note that ethical consideration existed long before the spread of AI based technologies. It is intrinsically tied to the field.
```

> The London Hospital survival predictor is a great example of that. It was created in 1972 and its job was to predict whether patients would recover from a coma. It had a dial that indicated either "survive" or "irreversible brain damage". It was one of the first applications of pattern recognition. Most surprisingly, it was one of the early forms of neural networks. Even at its birth, this raised concerns and doctors were advised to not take its input solely. The machine was never used to remove a patient's life support.

Reviewer: ```suggestion
The London Hospital survival predictor is a great example of that. It was created in 1972 to predict whether patients would recover from a coma. It had a dial that indicated either "survive" or "irreversible brain damage". It was one of the first applications of pattern recognition. Most surprisingly, it was also one of the early forms of neural networks. Even at its birth, this raised concerns and doctors were advised to not take its input solely. The machine was never used to remove a patient's life support.
```
